client/Bootstrap/Navigation/index.ts

```diff
@@ -0,0 +1,5 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+export * from './useRouteConfig/useRouteConfig.hook';
```

Comment:
This can change in a latter PR pending discussion - but would we want to expose the hook to anyone? I think this would be private, and thus not available via the barrel file

Reply:
That depends on how we feel about importing stuff even if private? An import from `Navigation` would still be quite nice, even if that component is also in navigation.

Reply:
I think it would be confusing. Public pieces only should be in the barrel - as the exposed API.

As an example, the server imports private pieces like this: https://github.com/strimzi/strimzi-ui/blob/5e277d42aa32e734f416347500b4b773df73f260/server/client/router.ts#L7 - it isnt that awkard/describes what is being imported and from where.